src/manifest.js

```diff
@@ -1,35 +1,54 @@
-import C2PAValidationError from "./errors/C2PAValidationError";
-import JumbBox from "./jumbBox";
-import { HardBindingAssertions } from "./validators/assertionValidator"
+import { JumbBox } from "./jumbBox.js";
 
-export default class Manifest {
+export class Manifest {
     manifestRoot = null;
     manifestType;
 
     constructor(jumbBox) {
         let boxWrapper = new JumbBox(jumbBox)
         this.manifestRoot = boxWrapper;
-        this.manifestType = boxWrapper.getRootDescriptionBox().content.uuidString;
+        this.manifestType = boxWrapper.descriptionBox.content.uuidString;
     }
 
     resolveClaim() {
-        let boxes = this.manifestRoot.resolveUuidString("c2cl");
+        let boxes = this.manifestRoot.resolveContentBoxesByUuid("c2cl");
         return boxes[0].content;
     }
 
-    resolveContentAssertion(trackName) {
-        let assertionBoxes = this.manifestRoot.resolveLabel('c2pa.assertions');
+    resolveFirstAssertion(...assertionNames) {
+        let assertions = this.resolveAssertions(...assertionNames);
+
+        if (assertions != null && assertions.length > 0) {
+            return assertions[0];
+        }
+
+        return null;
+    }
+
+    resolveAssertions(...assertionNames) {
+        let assertionBoxes = this.manifestRoot.resolveContentBoxesByLabel('c2pa.assertions');
         let jumbBoxes = assertionBoxes.map(x => new JumbBox(x));
+        let result = [];
 
         for (const box of jumbBoxes) {
-            for (const assertion of HardBindingAssertions) {
-                let result = box.resolveLabel(assertion);
-                if (result !== null) {
-                    return result[0].content;
+            for (const assertionName of assertionNames) {
+                let assertions = box.resolveContentBoxesByLabel(assertionName);
+                if (assertions !== null) {
+                    result.push(assertions[0].content);
                 }
             }
         }
 
-        throw new C2PAValidationError(`[C2PA] [${trackName}]Unable to resolve hard binding to content assertion from manifest.`);
+        return result;
+    }
+
+    resolveContainerByUri(uri) {
+        let prefix = "self#jumbf=c2pa/"
```

Comment:
I would move `"self#jumbf=c2pa/"` to the constant and place at the top of the document.